src/Lecture3.hs

```diff
@@ -134,11 +152,15 @@ data List1 a = List1 a [a]
 
 -- | This should be list append.
 instance Semigroup (List1 a) where
+  (<>) :: List1 a -> List1 a -> List1 a 
+  List1 x xs <> List1 y ys = List1 x (xs ++ (y :ys))
 
 
 {- | Does 'List1' have the 'Monoid' instance? If no then why?
 
 instance Monoid (List1 a) where
+
+  I wasn't able to find
```

Comment:
That's a possible answer! 😅 
Another simple answer is that there's no such thing as "empty non-empty list" 🙂